src/js/configs.js

```diff
@@ -113,12 +113,12 @@ module.exports = {
   nginx: {
     checked: true,
     highlighter: 'nginx',
-    latestVersion: '1.17.7',
+    latestVersion: '1.26.0',
     name: 'nginx',
     tls13: '1.13.0',
   },
   openssl: {
-    latestVersion: '1.1.1k',
```

Comment:
Latest openssl is 3.3.2.  The OpenSSL Software Foundation Inc currently supports openssl 3.x and later.  1.1.1 and earlier are EOL from the OpenSSL Software Foundation Inc (though Premium Support (paid) may still be available for 1.1.1w).  https://openssl-library.org/post/2023-09-11-eol-111/

Generating config for openssl < 3.0 might be something to consider for an "older" snapshot of the site, but most popular modern distros all ship openssl 3.x and so we should be producing configs for currently support versions of openssl.

Reply:
I am going to resolve this here since it is an openssl comment, and not an nginx comment.  There should be a separate issue opened for updating the openssl latest version to 3.3.2 after a review that existing directives recommended for apps using openssl are compatible with openssl 3.x.

Reply:
There are unfortunately known issues with OpenSSL 3 compatibility, so we're far from any official support at the moment.

Reply:
Could you point me at any documented compatibility issues with respect to ssl-config-generator?  I am initimately familiar with openssl 3.x interface changes and deprecations, and as I mentioned above, most *modern* popular distros ship the supported openssl 3.x instead of earlier versions.

Reply:
Yes, mainly not fiddling with dhparam and leaving that to the lib to infer the correct size. That however needs the server to either support that, or allow configuring the context to set that to auto. Which depends case by case, so may need gathering a bit of feedback for a while. (Servers relying on the new api might throw deprecation warnings for using the 1.1.1 route etc., therefore before any label reads "3.x.x" as the supported version, the configs must actually be 3+ ready at that time;)… hence why I kept it at the 1.1.1 branch, where no breaking changes in OP value defaults interfering with the applicability of the configs etc. happen — not to make any false promises for now.)

The other issue would be explaining how to support `old` config if one wishes, as TLS1–1.1 now need SECLEVEL=0:
> "The security strength of SHA1 and MD5 based signatures in TLS has been reduced. This results in SSL 3, TLS 1.0, TLS 1.1 and DTLS 1.0 no longer working at the default security level of 1 and instead requires security level 0."

I should probably open a meta/tracking issue for that actually, to keep it in one place.

Reply:
We should probably open two issues: one for openssl, and the other for following up with mozilla to refresh the specification recommendations.  For ssl-config-generator to remain useful it must support openssl 3.x.  Otherwise, will become an historical curiosity or merely a reference for outdated, less secure systems.

Reply:
#238 already quotes the suggestion to avoid specifying dhparams with openssl 3.

https://openssl-library.org/post/2024-09-10-tls-deprecation/ reminds us that the security level must be reduced to 0 for TLS1.0 and TLS1.1, and that support will also (likely) need to be explicitly selected at build time for openssl 3.5.